code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -703,23 +702,28 @@ var/global/list/damage_icon_parts = list()
 			var/obj/item/clothing/suit/S = wear_suit
 			var/image/bloodsies = image("icon" = species.blood_mask, "icon_state" = "[S.blood_overlay_type]blood")
 			bloodsies.color = wear_suit.blood_color
-			standing.overlays	+= bloodsies
+			standing.overlays += bloodsies
 
-		// Accessories - copied from uniform, BOILERPLATE because fuck this system.
 		var/obj/item/clothing/suit/suit = wear_suit
 		if(istype(suit) && suit.accessories.len)
 			for(var/obj/item/clothing/accessory/A in suit.accessories)
 				standing.overlays |= A.get_mob_overlay()
 
-		overlays_standing[SUIT_LAYER]	= standing
+		if(back && istype(back, /obj/item/weapon/rig))//and the main part also needed
+			var/obj/item/weapon/rig/R = back//this is safecheck, again
+			if(R && R.active)//we are fully engaged and charged
```

Comment:
don't some modules have overlays when not active too?

Reply:
BTW.
That's check rig activation not modules.